pkg/clusterapi/extraargs.go

```diff
@@ -62,6 +63,17 @@ func SecureEtcdTlsCipherSuitesExtraArgs() ExtraArgs {
 	return args
 }
 
+func NodeLabelsExtraArgs(wnc []v1alpha1.WorkerNodeGroupConfiguration) ExtraArgs {
+	args := ExtraArgs{}
+	for _, n := range wnc {
```

Comment:
I am assuming this for loop is for when we add multiple worker node group support, but when we do support that, would we be having multiple `node-labels` extra args per worker node group configuration appended in this manner? How would the difference in configurations be reflected in the capi spec template? For now, I would say to even forgo this loop here and operate under the assumption of one worker node group configuration, and solve that problem when we added multiple group support.

Reply:
The current design for multiple worker node groups is for each additional worker node group to have its own kubeadm config appended to the capi spec template, so each one will have its own `node-labels` arg. But I agree that since it's not implemented yet, we should still operate as if it's only one. I'll update it!